docs/UserManuals/ConfigUI/GitHub.md

```diff
@@ -77,12 +84,17 @@ Each GitHub repo has at most ONE set of transformation rules.
 
 #### Additional Settings (Optional)
 
+![image-20221103121859001](https://user-images.githubusercontent.com/3294100/199653539-7fbbd76e-512f-480b-ba3c-066742ed3c4a.png)
+
+- PR Body Pattern: Extract the issue numbers closed by pull requests. The issue numbers are parsed from PR bodies that meet the following RegEx. For Example:
```

Comment:
This configuration is not additional setting any more. It's now called 'PR-Issue Mapping'. See pic

![image](https://user-images.githubusercontent.com/14050754/199666530-7142d070-c373-4fbf-9222-cd4ee0a06568.png)